pkg/agent/run.go

```diff
@@ -184,37 +184,61 @@ func gatherData(ctx context.Context, config Config) []*api.DataReading {
 		dataGatherers[dgConfig.Name] = dg
 	}
 
-	// Fetch from all datagatherers
-	now := time.Now()
+	//TODO Change backoff parameters to those desired
+	backOff := backoff.NewExponentialBackOff()
+	backOff.MaxElapsedTime = 30 * time.Second
```

Comment:
This should come from a CLI flag with a default value.

Reply:
Good idea, what would you suggest for a default max elapsed time, and also default interval?

Reply:
I've set the default to 60 seconds for now